node-server/src/routing/routing.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
+import greetings from '../controllersMiddleware/greetings';
```

Comment:
We can just rename the path to something more clear:
```suggestion
import greetings from '../controllers/greetings';
```